BMSChecker_helperFun.cpp

```diff
@@ -0,0 +1,19 @@
+#include "BMSChecker.hpp"
+
+bool isRangeOk(float value, float LOWER_LIMIT, float HIGH_LIMIT) {
+	if (value < LOWER_LIMIT || value > HIGH_LIMIT) {
+		return false;
+	}
+	return true;
+}
+void printMessage() {
+	cout << " Out of Range!\n";
+	cout << " Au�er Reichweite!\n";
+}
+void printTolMessage() {
+	cout << "Early warning!: Reached  5% of Tolerance \n";
```

Comment:
Here it is explicitly mentioned that tolerance val is 5%. Whereas, in code a constant value is used.
This message loses its relevance if tolerance val is changed. Please use the constant everywhere, where tolerance value is used.